testsuite/stm32f3/openocd.cfg

```diff
@@ -0,0 +1,2 @@
+source [find interface/stlink.cfg]
```

Comment:
I'd been running `openocd` for this board probably per the Rust Embedded books using:
```
$ openocd -f interface/stlink-v2-1.cfg -f target/stm32f3x.cfg
```
but this gives:
```
WARNING: interface/stlink-v2-1.cfg is deprecated, please switch to interface/stlink.cfg
```
so per that warning updated to `interface/stlink.cfg` instead